LinterCopAnalyzers.resx

```diff
@@ -147,4 +147,13 @@
   <data name="Rule0003DoNotUseObjectIDsInVariablesOrPropertiesTitle" xml:space="preserve">
     <value>Do not use an Object ID for properties or variables declaration.</value>
   </data>
+  <data name="Rule0004LookupPageIdAndDrillDownPageIdDescription" xml:space="preserve">
+    <value>Property "LookupPageID" and "DrilldownPageID" must be filled in table because it is used in list page</value>
+  </data>
+  <data name="Rule0004LookupPageIdAndDrillDownPageIdFormat" xml:space="preserve">
+    <value>Property "LookupPageID" and "DrilldownPageID" must be filled in table "{1}" because it is used in page "{2}" (list)</value>
```

Comment:
Does this work? I thought the parameters are 0 based

Reply:
Must test it with an 0 based index... starting with 1 was successful (see screenshot below)
![image](https://user-images.githubusercontent.com/88825803/134668173-5f628862-0929-44a2-ac95-00315dd1fabf.png)

Got a bit confused with text constants of (C/)AL (%1 %2 ...)

Reply:
Ok interesting:

When starting at 0 following happens:

![image](https://user-images.githubusercontent.com/88825803/134668873-a5f56810-ab1c-41f3-ace2-39ec81a7816b.png)

Because of following:

![image](https://user-images.githubusercontent.com/88825803/134668996-bee1fcfd-ce28-4058-9b14-da197849052b.png)

You can't remove the "GetDeclaration" part, if you do you get following:

![image](https://user-images.githubusercontent.com/88825803/134669125-5ffa7a1a-a342-4e55-9e10-633958dfcbee.png)


So:
Starting the count of placeholder at {1} seems right